.github/workflows/release.yaml

```diff
@@ -195,3 +195,16 @@ jobs:
           title: "Return to development"
           body: Release complete for ${{github.ref_name}}
           token: ${{ steps.generate_token.outputs.token }}
+      - name: trigger npm publish
+        uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.actions.createWorkflowDispatch({
+              owner: context.owner,
+              repo: "buf-npm-publish",
+              workflow_id: "npm-publish.yaml",
+              ref: "main"
+              inputs: {
+                version: "${{env.NEXT_VERSION}}"
```

Comment:
`NEXT_VERSION` contains no `v` prefix, is it required?

Reply:
Nope, this action works on the version without `v`.